packages/datatrak-web/src/features/Questions/GeolocateQuestion/LatLongFields.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React, { ChangeEvent } from 'react';
+import styled from 'styled-components';
+import { TextInput } from '../../../components';
+
+const Wrapper = styled.div`
+  display: flex;
+  padding-bottom: 0.5rem;
+  .MuiFormControlLabel-root {
+    width: 6.3rem;
+    &:first-child {
+      margin-right: 1.125rem;
+    }
+  }
+`;
+
+export const LatLongFields = ({ setGeolocation, geolocation, name }: any) => {
+  const handleChange = (e: ChangeEvent<{}>, field: string) => {
+    setGeolocation({
+      ...geolocation,
+      [field]: parseFloat((e.target as HTMLInputElement).value),
```

Comment:
```suggestion
      [field]: parseFloat((e.target as HTMLInputElement).value),
      accuracy: 20 // hard code the accuracy to 20 for browser based location accuracy
```

According to the ticket description the data needs to match the data of the meditrak-app geo locate question which includes an accuracy value. The browser doesn't give us one but in the meditrak app code it has a comment which says that the target accuracy should be 20 so that seems like as good of default value as any.